src/platform/plugins/shared/controls/public/controls/timeslider_control/get_timeslider_control_factory.tsx

```diff
@@ -48,27 +45,39 @@ const displayName = i18n.translate('controls.timesliderControl.displayName', {
   defaultMessage: 'Time slider',
 });
 
-export const getTimesliderControlFactory = (): ControlFactory<
+export const getTimesliderControlFactory = (): EmbeddableFactory<
   TimesliderControlState,
   TimesliderControlApi
 > => {
   return {
     type: TIME_SLIDER_CONTROL,
-    getIconType: () => 'search',
-    getDisplayName: () => displayName,
-    buildControl: async ({ initialState, finalizeApi, uuid, controlGroupApi }) => {
+    buildEmbeddable: async ({ initialState, finalizeApi, uuid, parentApi }) => {
+      const state = initialState.rawState;
       const { timeRangeMeta$, formatDate, cleanupTimeRangeSubscription } =
-        initTimeRangeSubscription(controlGroupApi);
+        initTimeRangeSubscription(parentApi);
       const timeslice$ = new BehaviorSubject<[number, number] | undefined>(undefined);
-      const isAnchored$ = new BehaviorSubject<boolean | undefined>(initialState.isAnchored);
+      const isAnchored$ = new BehaviorSubject<boolean | undefined>(state.isAnchored);
```

Comment:
The play button isn't working as expected when achored:


- **Expected:**
   ![Oct-20-2025 08-38-40](https://github.com/user-attachments/assets/a08f2d55-d116-4bf1-8c0f-75a5f33a6a6f)
- **Actual:**
   ![Oct-20-2025 08-39-46](https://github.com/user-attachments/assets/f5fe9d11-e92f-41f7-af0b-36e9f5bdb28e)